CMakeLists.txt

```diff
@@ -86,7 +86,7 @@ set(IGN_MSGS_VER ${ignition-msgs8_VERSION_MAJOR})
 
 #--------------------------------------
 # Find SDFormat
-ign_find_package(sdformat12 REQUIRED VERSION 12.4.0)
+ign_find_package(sdformat12 REQUIRED VERSION 12.5.0)
```

Comment:
Don't forget to update the dependency on https://github.com/gazebo-release/gz-sensors6-release also

Reply:
oh thanks for the reminder. Here's the PR: https://github.com/gazebo-release/gz-sensors6-release/pull/9